exonum_client/crypto.py

```diff
@@ -28,7 +29,8 @@ class _FixedByteArray:
 
     def __init__(self, data: bytes, expected_len: int):
         if len(data) != expected_len:
-            raise ValueError("Incorrect data length (expected {}, got {}".format(expected_len, len(data)))
+            logging.critical(f"Incorrect data length: expected {expected_len}, got {len(data)}.")
+            raise ValueError("Incorrect data length: expected {}, got {}".format(expected_len, len(data)))
```

Comment:
Can we create one message and use it for logging and raising an exception ?

Reply:
Or you can create custom class for exception with `logging.critical(...` in `__init__` ?

Reply:
I think using one message is better.